git-extra/git-update-git-for-windows

```diff
@@ -174,7 +174,22 @@ update_git_for_windows () {
 		;;
 	esac
 
-	latest_tag_url=https://gitforwindows.org/latest-tag.txt
+	if test -f "$0.config"
+	then
+		fork="$(git config -f "$0.config" update.fromFork)"
+		test -n "$fork" || {
+			echo "Could not find update.fromFork in $0.config" >&2
```

Comment:
I get that end users shouldn't edit this config, but maybe we should fall back to the same behaviour we use when $0.config doesn't exist.

Wether the file exists or not, it's basically an unset config option.

Reply:
True. I now treat a `.config` file without `update.fromFork` the same as if the `.config` file was absent.